### DISCUSSION TITLE
Renaming LogEntry

### DISCUSSION THREAD

**Discussion by almirsarajcic** [3Stones-io/omedis#240] [General]
One idea was TimeTracker, but as @wintermeyer said, it doesn't make sense in the plural.

Some more ideas:
tracked_activities
activity_log_entries
...
Please contribute.

Or maybe leave the resource as it is?

**Comment by kagure-nyakio** [comment]
What about TimeSheet?

**Reply by wintermeyer** [reply]
AFAIK a TimeSheet could have many log entries but would never be a single entry. But English is not my native language.
